Presentation.Web/Models/GDPR/DataProcessingRegistrationDTO.cs

```diff
@@ -16,5 +18,9 @@ public DataProcessingRegistrationDTO(int id, string name)
         public NamedEntityWithEnabledStatusDTO[] ItSystems { get; set; }
 
         public ShallowOrganizationDTO[] DataProcessors { get; set; }
+
+        public YesNoIrrelevantOption? IsAgreementConcluded { get; set; }
```

Comment:
Jeg tænker om vi skal have os en ValueOptionWithDateDTO<T> 

Så fanger vi det begreb for det går igen flere steder.

Eksempel:
```
class ValueOptionWithDateDTO<T> {
   public T Value {get;set;}

  public DateTime? OptionalDateValue {get;set;}
}
```